File: packages/common/logs.js
module.exports =  
{
    INFO: function(message) {
        console.log("\x1b[92m[INFO] \x1b[97m" + message + " \x1b[39m")
    },

    WARNING: function(message) {
        console.log("\x1b[38;5;214m[WARNING] \x1b[97m" + message + " \x1b[39m")
    },

    ERROR: function(message) {
        console.log("\x1b[91m[ERROR] \x1b[97m" + message + " \x1b[39m")
    }
}

File: dotnet/resources/mygamemode/Logs.cs
using System;

namespace mygamemode
{
    public class Logs
    { 
        public static void INFO(String message) {
            Console.WriteLine($"[INFO] {message}");
        }
        
        public static void WARNING(String message) {
            Console.WriteLine($"[WARNING] {message}");
        }

        public static void ERROR(String message) {
            Console.WriteLine($"[ERROR] {message}");
        } 
    }
}